src/app/app-routing.module.ts

```diff
@@ -162,6 +149,20 @@ export const routes: Routes = [
       title: 'Areas'
     }
   },
+
+  // App Launcher
+  {
+    path: ':entity/:space/applauncher',
+    resolve: {
+      context: ContextResolver,
+    },
```

Comment:
There is a missing line? It should look like this:

resolve: {
  context: ContextResolver,
  featureFlagConfig: FeatureFlagResolver
},

Reply:
hmm - thought I merged this properly, but I've restored as requested.